src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -328,15 +330,16 @@ function MoneyRequestView({
     );
 
     const shouldShowMapOrReceipt = showMapAsImage || hasReceipt;
-    const shouldShowReceiptEmptyState = !hasReceipt && !isInvoice && (canEditReceipt || isAdmin || isApprover);
+    const isReceiptAllowed = !isPayReport && !isInvoice;
+    const shouldShowReceiptEmptyState = isReceiptAllowed && !hasReceipt && !isApproved && !isSettled && (canEditReceipt || isAdmin || isApprover);
```

Comment:
@Krishna2323 I believe we should show Receipt Header and Empty state only for paid polices as we are doing the same in `shouldShowNotesViolations` else it will be quite confusing.

Reply:
@Krishna2323 Can the receipt be edited by the other user(in P2P IOU) or admin in unpaid policy? If yes, we should leave the show empty state?

Reply:
Bump @Krishna2323

Reply:
In that case, I don't think we should show empty state as we are going to use it to allow user to add receipt. To be fair, the thumbnail represents Add Receipt not Empty Receipt.